doc/project/checklist.md

```diff
@@ -184,6 +177,20 @@ Any other software-visible design changes have been reviewed by the software tea
 
 All known "Won't Fix" bugs and "Errata" have been reviewed by the software team.
 
+### SEC_CM_IMPLEMENTED
+
+Any appropriate security counter-measures are implemented.
+For redundantly encoded FSMs, [the sparse-fsm-encode.py script](https://github.com/lowRISC/opentitan/blob/master/util/design/sparse-fsm-encode.py) must be used to generate the encoding.
```

Comment:
I think this was placed here to make it discoverable from the docs.

Long term, it would probably be good to link the secure design guidance doc by @cdgori somewhere here instead (this doc should come soon).
This FSM script (and related ones) should then be discoverable via that design guidance doc.

Reply:
Ok I'll leave that for now and we can update to link all these things to the guidance doc once available

Reply:
Perhaps we could list a few examples of countermeasures as one-liners to provide a faster context to the reader and then point to the guidance doc for more details.

Reply:
I have another 2 countermeasures: one-hot coding commands in aes and keymgr, hardened counter in keymgr and flash. Perhaps add them as one-liners here.